.gitignore

```diff
@@ -132,4 +132,8 @@ dmypy.json
 .pytype/
 
 # Cython debug symbols
-cython_debug/
\ No newline at end of file
+cython_debug/
+
+## Custom
+.idea/
+fasta_converted_from_sam_file.fasta
```

Comment:
Also added this so that the result file is not version-controlled. A better solution would be to write all output files to some directory `results/` or something, then add the entire directory `results/` as a pattern. If there are no version-controlled files in that directory, you can add a file `.gitkeep` in there, otherwise that directory itself won't be added to the index and then you would always have to make sure that the directory actually exists (or create it in your code) whenever you run your scripts.